16.cpp

```diff
@@ -0,0 +1,41 @@
+// https://leetcode-cn.com/problems/3sum-closest/
+#include <limits.h>
+
+#include <algorithm>
+#include <iostream>
+#include <vector>
+
+class Solution {
+ public:
+  int threeSumClosest(std::vector<int>& nums, int target) {
+    sort(nums.begin(), nums.end());
+    int last_num = 999;
+
+    int len = nums.size();
+    int ans = nums[0] + nums[len - 1] + nums[len - 2];
+    int min_diff = abs(target - ans);
```

Comment:
I sugest, rewrite as 
`int min_diff = std::numeric_limits<int>::max();`
this way, the code is more clean.